packages/pytest-simcore/src/pytest_simcore/helpers/utils_docker.py

```diff
@@ -14,6 +15,18 @@
 from tenacity.stop import stop_after_attempt
 from tenacity.wait import wait_fixed
 
+
+# SEE https://docs.docker.com/engine/api/v1.42/#tag/Container/operation/ContainerList
+class ContainerStatus(str, Enum):
```

Comment:
question: why don't you use the generated models in models-library?

Reply:
true